tools/clang/unittests/HLSLTestLib/DxcTestUtils.cpp

```diff
@@ -347,8 +347,8 @@ void WideToBlob(dxc::DxcDllSupport &dllSupport, const std::wstring &val,
 }
 
 void VerifyCompileOK(dxc::DxcDllSupport &dllSupport, LPCSTR pText,
-                     LPWSTR pTargetProfile, LPCWSTR pArgs,
-                     IDxcBlob **ppResult) {
+                     LPCWSTR pTargetProfile, LPCWSTR pArgs,
+                     _Outptr_ IDxcBlob **ppResult) {
```

Comment:
Why is `_Outptr_` being added back here, after having been removed from an earlier change?

Reply:
Same comment re: header and other overload.  I suspect it was a bad merge with that change.

Reply:
Fixed.